SUMMARY.md

```diff
@@ -6,6 +6,9 @@
 
 * [5-Minute Guide](get-started/5-minute-guide.md)
 * [Start Building](get-started/start-building.md)
+* [Choose Authentication Approach](get-started/authentication-approach/README.md)
+  * [Token-based (Native mobile or Single-page app)](get-started/authentication-approach/token-based.md)
+  * [Cookie-based (Website or Single-page app)](get-started/authentication-approach/cookie-based.md)
```

Comment:
No need to add these back, they were removed before intentionally. 
For broken links, please redirect to /get-started/start-building. It was written to replace these.